components/style/stylesheets/supports_rule.rs

```diff
@@ -129,20 +129,16 @@ impl SupportsCondition {
         match input.next()? {
             Token::ParenthesisBlock => {
                 input.parse_nested_block(|input| {
-                    // `input.try()` not needed here since the alternative uses `consume_all()`.
-                    parse_condition_or_declaration(input).or_else(|_| {
-                        consume_all(input);
-                        Ok(SupportsCondition::FutureSyntax(input.slice_from(pos).to_owned()))
-                    })
-                })
+                    input.try(|i| parse_condition_or_declaration(i).map(|_| ()))
```

Comment:
`.map(|_| ())` here drops a `SupportsCondition::Parenthesized` or `SupportsCondition::Declaration`, and replaces it with `()` that is eventually replaced with `SupportsCondition::FutureSyntax`. Is that the desired behavior?